mmv1/third_party/terraform/tests/resource_compute_instance_test.go

```diff
@@ -4406,6 +4441,52 @@ resource "google_compute_instance" "foobar" {
 `, disk, family, family, instance)
 }
 
+func testAccComputeInstance_network_performance_config(disk string, image string, instance string) string {
+	return fmt.Sprintf(`
+data "google_compute_image" "my_image" {
+  family  = "debian-9"
+  project = "debian-cloud"
+}
+
+resource "google_compute_disk" "foobar" {
+  name  = "%s"
+  zone  = "us-central1-a"
+  image = data.google_compute_image.my_image.self_link
+}
+
+resource "google_compute_image" "foobar" {
+  name        			= "%s-1"
+  source_disk 			= google_compute_disk.foobar.self_link
+  guest_os_features {
+    type = "GVNIC"
+  }
+}
+
+resource "google_compute_instance" "foobar" {
+  name         = "%s"
+  machine_type = "n2-standard-32"
```

Comment:
Can we use a smaller machine type here to avoid quota issues? I think our tests don't have anything higher than `n2-standard-2`

Reply:
I wish I could and actually I needed to raise quota in my project to test this also but the configuration only applies (from what I can tell) to machines with 32+ vCPUs. The [compatible types are on this docs page](https://cloud.google.com/compute/docs/networking/configure-vm-with-high-bandwidth-configuration#general_purpose_n2_machine_types).

I suppose if the configuration takes hold on a smaller instance type and has no practical egress bandwidth effect, would that be the preferred scenario to test?

Reply:
Yes let's see if the API accepts the field on a smaller type and go with that if possible. If not, then yeah it looks like we'll have to raise quota on our test project. Let me know!